src/data/resolvers/motions.ts

```diff
@@ -920,15 +921,103 @@ export const motionsResolvers = ({
       const actionValues = colonyClient.interface.parseTransaction({
         data: action,
       });
+
+      // PaymentMotion
+      if (!actionValues) {
+        const oneTxPaymentClient = await colonyManager.getClient(
+          ClientType.OneTxPaymentClient,
+          colonyAddress,
+        );
+
```

Comment:
its quite a big assumption, because we cant be sure that if there is no actionValues above it means that its PaymentMotion. Did you check if there is any other way to find out if this is actually PaymentMotion? what action include?

Reply:
Hmmm, from what I saw I don't remember seeing something that could indicate if this is a PaymentMotion other than the actionValues coming null/undefined but I'll check again